### PR TITLE
allow to sync olm operators

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

**Comment by karmab** [comment]
@yuvalk can you review?

File: 065_olm.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# Variables to set, suit to your installation
+cd /root
+export PATH=/root/bin:$PATH
+export OCP_RELEASE="{{ disconnected_operators_version|default(openshift_version|default(4.7)) }}"
+export OCP_PULLSECRET_AUTHFILE='/root/openshift_pull.json'
+IP=$(hostname -I | awk -F' ' '{print $2}')
+REVERSE_NAME=$(dig -x $IP +short | sed 's/\.[^\.]*$//')
+echo $IP | grep -q ':' && SERVER6=$(grep : /etc/resolv.conf | grep -v fe80 | cut -d" " -f2) && REVERSE_NAME=$(dig -6x $IP +short @$SERVER6 | sed 's/\.[^\.]*$//')
+REGISTRY_NAME=${REVERSE_NAME:-$(hostname -f)}
+export LOCAL_REGISTRY=$REGISTRY_NAME:5000
```
Comment (yuvalk) [inline 065_olm.sh:11]: so it have to be 5000?

Comment (yuvalk) [reply 065_olm.sh:11]: also LOCAL here is misleading, actually ppl can run the registry externally to the installer, right (as long as they provide the access info)

File: 065_olm.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+export LOCAL_REGISTRY_INDEX_TAG=olm-index/redhat-operator-index:v$OCP_RELEASE
```
Comment (yuvalk) [inline 065_olm.sh:12]: does this means the catalog will be called redhat-operators too?
(so you can then use regulard subscriptions)

kudos if so

File: 065_olm.sh (hunk 1 above)
Comment (karmab) [reply 065_olm.sh:11]: the plan always uses local registry

Comment (yuvalk) [reply 065_olm.sh:11]: yeah, I'd say, that's a missing feature (to support external disconnected registry)

**Pull request merged**